storage/database/schema.go

```diff
@@ -270,3 +276,31 @@ func TrieNodeKey(hash common.ExtHash) []byte {
 		return hash.Bytes()
 	}
 }
+
+type PruningMark struct {
+	Number uint64
+	Hash   common.ExtHash
+}
+
+// TriePruningMarkKey = prefix + number + hash
+// Block number comes first to sort the entries by block numbers.
+// Later we can iterate through the marks and extract any given block number range.
+func pruningMarkKey(mark PruningMark) []byte {
+	bNumber := make([]byte, 8)
```

Comment:
Considerably used the magic number 8 . How about declaring a dedicated constant?

Reply:
I tried to find an appropriate built-in constant for uint64 byte size but to no avail. There is an unaccepted feature request but not in standard packages: https://github.com/golang/go/issues/29982#issuecomment-572199921. Even golang PutUint64 uses hardcoded constant  https://github.com/golang/go/blob/go1.20.6/src/encoding/binary/binary.go#L184. Therefore, I shall keep it literal number 8.

Reply:
`const size = unsafe.Sizeof(uint64(0))` was not fit it? This alternative seems exactly the same behavior. I guess it seems not to fit your favorite, shall we skip it?

Reply:
Oh, unsafe should do the trick. But I think the variable name `bNumber` should give the necessary hint. Yes, let's skip it.